LOGGING/src/NLogTarget/ApplicationInsightsTarget.cs

```diff
@@ -118,10 +129,24 @@ protected override void InitializeTarget()
             this.telemetryClient = new TelemetryClient();
 #pragma warning restore CS0618 // Type or member is obsolete
 
-            string instrumentationKey = this.instrumentationKeyLayout.Render(LogEventInfo.CreateNullEvent());
-            if (!string.IsNullOrWhiteSpace(instrumentationKey))
+            string connectionString = this.connectionStringLayout.Render(LogEventInfo.CreateNullEvent());
```

Comment:
this is a good addition! Would need to update docs (if any), to callout the change, and that connection string will be preferred over ikey.

Reply:
https://github.com/microsoft/ApplicationInsights-dotnet/tree/main/LOGGING#nlog this doc

Reply:
@cijothomas - Updated the documentation can you please take a look on updated commits.

Reply:
@cijothomas - Can you please review the updated changes.